store.go

```diff
@@ -1749,6 +1729,23 @@ func (s *store) Container(id string) (*Container, error) {
 	return rcstore.Get(id)
 }
 
+func (s *store) ContainerLayerID(id string) (string, error) {
+	rcstore, err := s.ContainerStore()
+	if err != nil {
+		return "", err
+	}
+	rcstore.Lock()
+	defer rcstore.Unlock()
+	if modified, err := rcstore.Modified(); modified || err != nil {
+		rcstore.Load()
+	}
+	container, err := rcstore.Get(id)
+	if err != nil {
```

Comment:
Thought I saw a case where we could get a nil container w/o an error.  Just ran through the code again and you're right, the container shouldn't be nil if there's not an error.  Sorry about the fire drill.